src/Mesh/Topography.jl

```diff
@@ -0,0 +1,159 @@
+module Topography
+"""
+ module Topography
+    
+"""
+
+using Printf
+using DelimitedFiles
+using Dierckx
+
+export TopographyReadExternal, ReadExternalHeader, ReadExternalTxtCoordinates
+
+import Canary
+using MPI
+
+
+# {{{ READTOPOtxt_header
+"""
+   READTOPOtxt_header(txt_inputfile,nlon, nlat, deltaLon, deltaLat)
+
+   reads the header file containing the details of the topography from a NOAA text file
+"""
+function ReadExternalHeader(header_file_in)
+    
+    @info @sprintf """ Topography header file %s ... DONE""" header_file_in
```

Comment:
Some of these `@info` lines seem more like `@debug`

Reply:
No, the user needs to know where the code is in terms of reading his/her grid file. The file that the user passes to the code may be broken and hence the would be hanging without knowing where it is hanging.